editor.planx.uk/src/@planx/components/Content/Public.tsx

```diff
@@ -25,10 +24,18 @@ const Content = styled(Box, {
   },
 }));
 
+Content.defaultProps = {
+  color: "#ffffff",
+};
+
 const ContentComponent: React.FC<Props> = (props) => {
   return (
     <Card handleSubmit={props.handleSubmit} isValid>
-      <Content color={props.color} data-testid="content">
+      <Content
+        color={props.color}
+        data-testid="content"
+        p={props.color === ("#ffffff" || undefined) ? 0 : 2}
```

Comment:
```suggestion
        p={(props.color === "#ffffff" || !props.color) ? 0 : 2}
```

Reply:
It's just a little syntax issue here - we need to check `props.color` against both conditions. I've shorthanded here using `!` but the above is equal to - 

```js
props.color === "#ffffff" || props.color === undefined ? 0 : 2
```

Reply:
Ah gotcha, I need to revisit the basics and brush up on my syntax.

Thanks for the help with this.